pymc3/distributions/timeseries.py

```diff
@@ -309,8 +311,30 @@ def _random(self, sigma, mu, size, sample_shape):
         else:
             axis = 0
         rv = stats.norm(mu, sigma)
-        data = rv.rvs(size).cumsum(axis=axis)
-        data = data - data[0]  # TODO: this should be a draw from `init`, if available
+        if size is None:
+            data = rv.rvs(size).cumsum(axis=axis)
+            data = data - data[0]  # TODO: this should be a draw from `init`, if available
+        elif len(size) == 1:
+            data = rv.rvs(size).cumsum(axis=axis)
+            data = data - data[0]  # TODO: this should be a draw from `init`, if available
```

Comment:
It looks like this could be simplified to:
```suggestion
        if (size is None) or (len(size) == 1):
            data = rv.rvs(size).cumsum(axis=axis)
            data = data - data[0]  # TODO: this should be a draw from `init`, if available
```